.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,25 @@
+## Details
+___Do not mention proprietary info or link to internal work items in this PR.___
+
+**What were the changes?**  
+_One sentence describing the work done._
+
+**Why were the changes made?**  
+_Explain the motivation behind the work. Provide any publicly-available historical context._
+
+**How was the outcome achieved?**  
+_Technical details behind the work. Explain any publicly-available hardware peculiarities._
+
+**Additional Documentation:**  
+_What else should the reviewer know?_
+
+**Test Cases:**
+- [ ] _List here_
+
+## Approval Checklist
+- [ ] Ensure above details are appropriately filled
+- [ ] Ensure relevant internal documentation has been updated
+- [ ] Verify appropriate developer testing was done
```

Comment:
Not sure how reviewer can "Verify appropriate developer testing was done"? Email author? I think this needs to be moved to checklist before merging. Author is responsible for this testing

Reply:
Actually, for this item, the idea is that the author should not have submitted a PR on untested work, so the approver should not approve unless that happened. The author is responsible for the testing but the approver is responsible not to approve unless the testing was done.

The author should be listing their test cases in the Details section, which is already covered by "Ensure above details are appropriately filled", so I will remove this.